src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -467,13 +467,25 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           source.setAttribute('type', mimetype);
 
           if (this._config.useMediaOptionAttribute) {
-            const options = {};
-            options.option = {};
+            let options = {};
+            // https://webostv.developer.lge.com/develop/guides/mediaoption-parameter
```

Comment:
what is the purpose of posting this URL here?

Reply:
this is the definition of possible configs